### DISCUSSION TITLE
Do you support UDP?

### DISCUSSION THREAD

**Discussion by sdanfa** [any1/wayvnc#315] [General]
Hello,
i'd like to know if the current wayvnc program supports UDP? It's needed for better performance on high latentcy networks especially.

Thanks!

**Comment by any1** [comment]
There is no open standard for how to use UDP for VNC. It looks like Real VNC have implemented some UDP communication, but they have not told anyone how it works.

I am also rather sceptical that it actually helps much. It's good for audio, where you can afford to lose a few packets, but for video you basically end up re-implementing TCP over UDP to make it reliable.

It would also be fine for input events (mouse, keyboard), but those events flow the other direction, so they're not affected by large video frames being sent from the server. Well, maybe if you have a half-duplex connection...
